README.md

```diff
@@ -1,17 +1,20 @@
-#Infrared
+# Infrared
+
+[![Code of Conduct](https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg?style=flat)](https://github.com/tessel/project/blob/master/CONDUCT.md)
+
 Driver for the ir-attx4 Tessel infrared module. The hardware documentation for this module can be found [here](https://github.com/tessel/hardware/blob/master/modules-overview.md#infrared).
 
 If you run into any issues you can ask for support on the [IR Module Forums](https://forums.tessel.io/c/modules/ir).
 
 Note: This library is responsible for sending and receiving IR data but we've just started another library, [ir-codes](https://github.com/technicalmachine/ir-codes), to generate and parse IR signals from different manufacturers. It's currently missing most manufacturers so we encourage developers to help us build out that library.
 
-###Installation
-```sh
+### Installation
+```shell
```

Comment:
Actually, I am new with Markdown Syntax so was just exploring the different aliases of shellscript like sh, shell, bash, zsh etc.